scripts/helmAcs.sh

```diff
@@ -256,7 +256,7 @@ share:
     initialDelaySeconds: 420
 registryPullSecrets: quay-registry-secret" > $VALUES_FILE
 
-  CHART_VERSION=1.1.8
+  CHART_VERSION=1.1.10
```

Comment:
Just a note that chart *1.1.10* is not in stable, just in incubator. We didn't release to stable for RC7. We can do that if required for testing.